nixos/modules/services/networking/harmonia.nix

```diff
@@ -74,8 +72,6 @@ in
         ProtectProc = "invisible";
         RestrictNamespaces = true;
         SystemCallArchitectures = "native";
-
-        PrivateNetwork = false;
```

Comment:
I would like to keep this to acknowledge that I know about this option but explicitly decided against.

Reply:
Everything else looks alright.